src/haz3lcore/statics/TypCtx.re

```diff
@@ -0,0 +1,17 @@
+include CtxBase;
+
+module Kind = {
+  [@deriving (show({with_path: false}), sexp, yojson)]
+  type t =
+    | Singleton;
```

Comment:
Singleton should take an argument

Reply:
It's just a stub, will be moved out as a separate module and actually complete it.